nailgun/entities.py

```diff
@@ -3913,6 +4110,7 @@ def __init__(self, server_config=None, **kwargs):
             'url': entity_fields.URLField(required=True),
             'location': entity_fields.OneToManyField(Location),
             'organization': entity_fields.OneToManyField(Organization),
+            'environment_id': entity_fields.IntegerField(),
```

Comment:
Are you sure that we have integer parameter `environment_id`? Because usually we are working with entities, not their ids